demos/common/export_models/export_model.py

```diff
@@ -0,0 +1,361 @@
+
+embedding_graph_template = """input_stream: "REQUEST_PAYLOAD:input"
+output_stream: "RESPONSE_PAYLOAD:output"
+node {
+  calculator: "OpenVINOModelServerSessionCalculator"
+  output_side_packet: "SESSION:tokenizer"
+  node_options: {
+    [type.googleapis.com / mediapipe.OpenVINOModelServerSessionCalculatorOptions]: {
+      servable_name: "{{model_name}}_tokenizer_model"
+    }
+  }
+}
+node {
+  calculator: "OpenVINOModelServerSessionCalculator"
+  output_side_packet: "SESSION:embeddings"
+  node_options: {
+    [type.googleapis.com / mediapipe.OpenVINOModelServerSessionCalculatorOptions]: {
+      servable_name: "{{model_name}}_embeddings_model"
+    }
+  }
+}
+node {
+  input_side_packet: "TOKENIZER_SESSION:tokenizer"
+  input_side_packet: "EMBEDDINGS_SESSION:embeddings"
+  calculator: "EmbeddingsCalculator"
+  input_stream: "REQUEST_PAYLOAD:input"
+  output_stream: "RESPONSE_PAYLOAD:output"
+  node_options: {
+    [type.googleapis.com / mediapipe.EmbeddingsCalculatorOptions]: {
+      normalize_embeddings: {{normalize|default("true", true)}}
+    }
+  }
+}
+"""
+
+rerank_graph_template = """input_stream: "REQUEST_PAYLOAD:input"
+output_stream: "RESPONSE_PAYLOAD:output"
+node {
+  calculator: "OpenVINOModelServerSessionCalculator"
+  output_side_packet: "SESSION:tokenizer"
+  node_options: {
+    [type.googleapis.com / mediapipe.OpenVINOModelServerSessionCalculatorOptions]: {
+      servable_name: "{{model_name}}_tokenizer_model"
+    }
+  }
+}
+node {
+  calculator: "OpenVINOModelServerSessionCalculator"
+  output_side_packet: "SESSION:rerank"
+  node_options: {
+    [type.googleapis.com / mediapipe.OpenVINOModelServerSessionCalculatorOptions]: {
+      servable_name: "{{model_name}}_rerank_model"
+    }
+  }
+}
+node {
+    input_side_packet: "TOKENIZER_SESSION:tokenizer"
+    input_side_packet: "RERANK_SESSION:rerank"
+    calculator: "RerankCalculator"
+    input_stream: "REQUEST_PAYLOAD:input"
+    output_stream: "RESPONSE_PAYLOAD:output"
+}
+"""
+
+text_generation_graph_template = """input_stream: "HTTP_REQUEST_PAYLOAD:input"
+output_stream: "HTTP_RESPONSE_PAYLOAD:output"
+
+node: {
+  name: "LLMExecutor"
+  calculator: "HttpLLMCalculator"
+  input_stream: "LOOPBACK:loopback"
+  input_stream: "HTTP_REQUEST_PAYLOAD:input"
+  input_side_packet: "LLM_NODE_RESOURCES:llm"
+  output_stream: "LOOPBACK:loopback"
+  output_stream: "HTTP_RESPONSE_PAYLOAD:output"
+  input_stream_info: {
+    tag_index: 'LOOPBACK:0',
+    back_edge: true
+  }
+  node_options: {
+      [type.googleapis.com / mediapipe.LLMCalculatorOptions]: {
+          models_path: "{{model_path}}",
+          plugin_config: '{"KV_CACHE_PRECISION": "{{kv_cache_precision|default("", false)}}"}',
+          block_size: {{block_size|default("32", true)}},
+          enable_prefix_caching: {{enable_prefix_caching|default("true", true)}},
+          cache_size: {{cache_size|default("10", true)}},
+          max_num_batched_tokens: {{max_num_batched_tokens|default("8120", false)}},
```

Comment:
The default for num batched tokens depends on scheduler setting (in dynamic split fuse we use smaller values)